docs/development/_index.md

```diff
@@ -66,11 +66,18 @@ make app-linux
 ```
 
 ```bash
-make app-windows
+make app-mac
 ```
 
 ```bash
-make app-mac
+make app-win
+```
+
+For Windows, by default it will produce an installer using [NSIS (Nullsoft Scriptable Install System)](https://sourceforge.net/projects/nsis/).
+If you prefer an `.msi` installer, then be sure to install the [WiX toolset](https://wixtoolset.org/), have its `light.exe` and `candle.exe` in the Windows path, and run:
+
+```bash
```

Comment:
```
set PATH=%PATH%;C:\Program Files (x86)\WiX Toolset v3.11\bin
make app-win-msi
```

Reply:
Would you mind adding this set PATH command in to the docs there?

An improvement for later might be to check if WiX Toolset is there and add it to the path inside the make target/or npm script.